lib/segment/src/id_tracker/simple_id_tracker.rs

```diff
@@ -405,6 +434,41 @@ impl IdTracker for SimpleIdTracker {
         }
         Ok(())
     }
+
+    fn make_immutable(&self, _save_path: &Path) -> OperationResult<IdTrackerEnum> {
+        // TODO: uncomment this when we integrate the new immutable id tracker
+        /*
+               let external_to_internal = self
+                   .external_to_internal_num
+                   .iter()
+                   .map(|(k, v)| (StoredPointId::NumId(*k), *v))
+                   .chain(
+                       self.external_to_internal_uuid
+                           .iter()
+                           .map(|(k, v)| (StoredPointId::Uuid(*k), *v)),
+                   )
+                   .collect();
+
+               let mappings = PointMappings {
+                   external_to_internal,
+                   internal_to_version: self.internal_to_version.clone(),
+                   internal_to_external: self.internal_to_external.iter().map(|i| i.into()).collect(),
+               };
+
+               let immutable_tracker =
+                   ImmutableIdTracker::new(save_path.to_path_buf(), self.deleted.clone(), mappings);
+
+               // Make it persistent
+               immutable_tracker.save()?;
+
+               Ok(IdTrackerEnum::ImmutableIdTracker(immutable_tracker))
+        */
```

Comment:
Rather than commented-out bits I'd prefer not having them here at all, and adding them in a separate commit (in a new PR).

Do you think that's possible without too much effort?

If that does take a lot of effort I'm fine with leaving them in this time.